packages/css/tests/index.test.ts

```diff
@@ -1183,4 +1183,61 @@ describe("createCss", () => {
       ./*X*/_fOVguX/*X*/./*X*/_fOVguX/*X*/./*X*/_fOVguX/*X*/./*X*/_fOVguX/*X*/./*X*/_fOVguX/*X*/./*X*/_fOVguX/*X*/:disabled{color:red;}"
     `);
   });
+
+  test("should handle global styles", () => {
+    const css = createCss({}, null);
+    const { styles } = css.getStyles(() => {
+      css
+        .global({
+          "@media (min-width: 700px)": {
+            div: {
+              color: "red",
+              backgroundColor: "white",
+              paddingLeft: "10px",
+            },
+          },
+        })
+        .toString();
+      return "";
+    });
+
+    expect(styles[1].trim()).toMatchInlineSnapshot(`
+      "/* STITCHES */
+      @media (min-width: 700px){ div{padding-left:10px;}}
+      @media (min-width: 700px){ div{background-color:white;}}
+      @media (min-width: 700px){ div{color:red;}}"
+    `);
+  });
+
+  test("should not re-inject global styles", () => {
+    const css = createCss({}, null);
+    const { styles } = css.getStyles(() => {
+      css
+        .global({
+          "@media (min-width: 700px)": { div: { color: "red" } },
+        })
+        .toString();
+
+      css
+        .global({
+          "@media (min-width: 700px)": { div: { color: "red" } },
+        })
+        .toString();
+      return "";
+    });
+
+    expect(styles[1].trim()).toMatchInlineSnapshot(`
+      "/* STITCHES */
+      @media (min-width: 700px){ div{color:red;}}"
+    `);
+  });
+    test("should error when styles are used without nesting", () => {
+    const css = createCss({}, null);
+    expect(()=> {
+      css.global({
+          background: 'red'
+        })
+    }).toThrow()
```

Comment:
Im not following this use case. Why do they need to be nested?

To be clear, we need to be able to [do this with the global CSS](https://github.com/modulz/radix/blob/master/packages/radix/src/GlobalStyles.tsx#L4-L19) function

The main reason for this is to remove styles from `body`, `html` and have access to tokens and breakpoints

Reply:
Because not having nesting at all doesn't make sense unless you're creating a class to attach the styles to and in this case the styles wouldn't be global (if they're not inside an explicit selector like body or HTML etc..)

```
   css.global({
          background: 'red'
        })
```

this code above can't be global since it lacks a selector and should error ^

Reply:
@hadihallak ahhh yeah, of course 😓 

I was thrown off by the wording. Can we be more explicit about the need of defining a selector?